extensions/GitExtension/FileExplorerGitIntegration/Models/GitRepositoryStatus.cs

```diff
@@ -45,6 +36,11 @@ public void Add(string path, GitStatusEntry status)
         }
     }
 
+    public bool TryAdd(string path, SubmoduleStatus status)

```

Comment:
May we consider using a more descriptive method name instead of TryAdd which is identical to the called Dictionary method name? Thanks!

Reply:
I deliberately chose `TryAdd` *because* it is identical to the method on `Dictionary`. My hope was that would make the semantics clear.